src/views/resultsView/createScanProvider.ts

```diff
@@ -153,8 +154,12 @@ export async function createScan(
   }
 
   updateStatusBarItem(constants.scanCreatePreparing, true, statusBarItem);
-  await createScanForProject(context, logs);
-
+  try {
+    await createScanForProject(context, logs);
+  } catch (error) {
+      setScanButtonDefaultIfScanIsNotRunning(context);
```

Comment:
what happen if scan is running?
maybe we want to add a log with this information to notify the user abut stack scan?

Reply:
The run scan button will be a cancel button.
the user get a notification with the error (I keep throwing the error)

Reply:
will method setScanButtonDefaultIfScanIsNotRunning work properly even if the button is cancel and not run?